api/api.yaml

```diff
@@ -0,0 +1,299 @@
+swagger: "2.0"
+info:
+  description: "This is a explorer service. It can be used to create, view, modify and delete explorers. It is also used to login and logout of the system."
+  version: "1.0.0"
+  title: "Explorer Service API"
+  contact:
+    email: "prashanth.ramakrishnan@gmail.com"
```

Comment:
update contact as below or maybe some other way that would be common across all the API's
`  contact:
    name: "Github Issues"
    url: "https://github.com/prambrucke/tafel/issues"`